sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -32,10 +32,18 @@ Licensed under the MIT License.
     <tag>HEAD</tag>
   </scm>
 
+  <repositories>
+    <repository>
+      <id>azuresdkmaven</id>
+      <url>https://azsdkartifacts.blob.core.windows.net/maven</url>
```

Comment:
@kushagraThapar is this the recommended approach by the central SDK?

Reply:
Yes this is given by central sdk